### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by namiyousef** [namiyousef/multi-task-learning#22]
Number of epochs used:
Batch_size used:
Model architecture:
- Encoder: Resnet34 pre-trained=False
- Decoders:
-- Class:
-- Segmen:
-- BB:  

### Train 
Configuration |  BB (L1Loss) | Class (BCELoss) | Segmen (DiceLoss) | Total
--- | --- | --- | --- | ---
Baseline | | | |
Baseline + Class | | | |
Baseline + BB | | | |
Baseline + BB + Class| | | |
Seg + bb (0.75, 0.25) |22.136 |- |0.125 |
Seg + bb (0.25, 0.75)| 20.152| -|0.122 |
seg + bb + class (0.5, 0.25, 0.25)|0.3001|1.0348 |0.1527|
seg + bb + class (0.25, 0.5, 0.25)|0.2744 |1.1820 | 0.1582|
seg + bb + class (0.25, 0.25, 0.5)|0.3564|0.77764|0.1979 |
seg + bb (uniform)| | | |
seg + bb (const. bern.)| | | |

### Test
Configuration |  BB (L1Loss) | Class (BCELoss) | Segmen (DiceLoss) | Total
--- | --- | --- | --- | ---
Baseline | -|- | 0.137|
Baseline + Class | -|1.278 | 0.182|
Baseline + BB | 21.254| - | 0.126|
Baseline + BB + Class|0.2892 |1.4871 |0.1698 |

### Train
Configuration | Class (Accuracy) 
--- | --- 
Baseline + Class | 
Baseline + BB + Class| 

### Test
Configuration | Class (Accuracy) 
--- | --- 
Baseline + Class | 
Baseline + BB + Class|
